### PR TITLE
Fixing tab order of header

### PR DESCRIPTION
**Pull request by christaylor19**
Commented out Twitter Auth secrets to get things working - lmk if I need to avoid that

I've got Prettier code formatter set up in my IDE so it's reformatting the code unfortunately. It's not the most readable PR, but it consists of changing the order of the items in the main header and removing the clas `c-nav__item--right` and adding `tabindex={0}`

### CODE REVIEW

**Review by BrunoSalerno — CHANGES_REQUESTED** [review]
Great job @christaylor19 ;), but I think that you should remove from the PR all changes unrelated to the issue (including the style-related ones), for two reasons: it's good to keep the rule 1 PR -> 1 issue (or change), and it makes the PR and the commits easier to read (and review!).
One easy way to do this could be performing a `checkout` to the original files in master, so you can chose exactly what to commit. For example:  
1. `git checkout master -- client/src/components/main.js`, to grab the original file from master, which will end up in staging.
2. Then `git reset client/src/components/main.js` to unstage the changes on that file.
3. And then `git add -p client/src/components/main.js` to chose what to put in staging.

**Comment by christaylor19** [comment]
@BrunoSalerno Fixed 👍 The issue is that I had Format on Save set up and also Prettier formatter which added a few bits automatically.

File: client/src/components/main.js [hunk 1]
```diff
@@ -80,55 +80,96 @@ class Main extends Component {
   }
 
   onAcceptCookie() {
-    BrowserCookies.set('cookie-advice-accepted', "yes", {expires: 365});
+    BrowserCookies.set('cookie-advice-accepted', 'yes', { expires: 365 });
```
Comment (BrunoSalerno) [inline client/src/components/main.js:83]: Here there are changes in the quote and spacing around brackets. As they are not related to the issue, they shouldn't be here either. Any changes in style of pre-existing code should be addressed in another PR.

File: client/src/components/main.js [hunk 2]
```diff
@@ -80,55 +80,96 @@ class Main extends Component {
...
     MainStore.hideCookieAdvice();
   }
 
   render() {
     return (
-        <div>
-          <nav className="c-nav c-nav--inline">
```
Comment (BrunoSalerno) [inline client/src/components/main.js:90]: Here there is modification in the indentation.

**Review by BrunoSalerno — CHANGES_REQUESTED** [review]
@christaylor19 I still can see changes in the indentation, spacing, quotes, and in other places such as `api/config/auth.rb`, and it even adds the `package-lock.json` file. I added comments in the code to show this to you.
What I meant in my original review is that the PR should only contain changes related to the issue, and nothing else.  This includes other files, but also the style or the code around the new code if it is not related to the issue. For example if you want to add a new method to a given class, you should not refactor the class; just the code related to the new method (the bare minimum). An example of this in this PR is `main.js`, where, among other things,  there is a change in the indentation. This should not be in the PR because it's not related to the issue, but in this case it also makes it very difficult to see the actual change.
The `package-lock.json` file should not be in the PR either, as this project uses yarn (the lock file is `yarn.lock`). Of course this could be modified, but we should discuss it first, and the change should be done in a separate PR. The idea of the lock file is to have a unique source of true regarding the version of the packages, so it would not make sense to have both either.

**Pull request closed without merging**